MapboxDirectionsTests/V5Tests.swift

```diff
@@ -52,6 +52,13 @@ class V5Tests: XCTestCase {
             XCTAssertEqual(task.state, .completed)
         }
         
+        let opts = route!.routeOptions
+        
+        XCTAssertEqual(opts.profileIdentifier, .automobile)
```

Comment:
Might as well assert that `route!.routeOptions` is equal to `options`. That way we can be sure they’re literally the same object; no need to compare individual properties on them.